master/getting-started/openstack/upgrade/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Upgrading Calico for OpenStack
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/openstack/upgrade/'
+---
+## Upgrade procedure (OpenStack)
+These documents detail the procedure for upgrading {{site.prodname}} in an OpenStack system. 
+If a release does not upgrade a particular component, you should skip the steps for that component.
+
+> **Important**: You will
+> be unable to issue API requests to your OpenStack system during the
+> procedure. Please plan your upgrade window accordingly, and see the
+> [Service Impact](#service-impact) section for more details.
+{: .alert .alert-danger}
+
+## Service impact
+
+During the upgrade, **all VMs will continue to function normally**:
+there should be no impact on the data plane. However, control plane
+traffic may fail at different points throughout the upgrade.
+
+Generally, users should be prevented from creating or updating virtual
+machines during this procedure, as these actions will fail. VM deletion
+*may* succeed, but will likely be delayed until the end of the upgrade.
+
+For this reason, we highly recommend planning a maintenance window for
+the upgrade. During this window, you should disable all user API access
+to your OpenStack deployment.
+
+
+## Before you begin
+
+- You must first [upgrade](/v2.6/getting-started/openstack/upgrade) 
+  to {{site.prodname}} [v2.6.x](https://github.com/projectcalico/calico/releases) 
+  before you can upgrade to {{site.prodname}} 
+  {{site.data.versions[page.version].first.title}}. 
+
+## Upgrade steps
+
+If you use calicoctl to add Calico policies in addition to those that are derived automatically (by our Neutron driver) from OpenStack security groups - or more generally to configure any Calico data other than the data that is generated by the Calico Neutron driver - you will need to follow the steps marked OPTIONAL below. Otherwise you can skip those OPTIONAL steps.
```

Comment:
I am not sure that this is the best presentation. At a high level, the steps marked OPTIONAL are not actually optional....they are required. 😄 So...that is an issue where you have the wrong words.

I would instead suggest that you hew more closely to the structure that I have in the Kubernetes upgrade docs (biased, I know!). LOL